src/lab4.py

```diff
@@ -0,0 +1,62 @@
+class Zoo:
+    def __init__(self, vs_number=0, name="", an_number=0, year=0, type_of_animal=""):
+        self.__vs_number = vs_number
+        self.__name = name
+        self.__an_number = an_number
+        # не приватні
```

Comment:
Everything is okay, but take away this comment, it is not needed here

Reply:
done